Formula/spidermonkey.rb

```diff
@@ -1,54 +1,17 @@
 class Spidermonkey < Formula
-  desc "JavaScript-C Engine"
+  desc "SpiderMonkey is Mozilla's JavaScript engine, as used in Firefox"
   homepage "https://developer.mozilla.org/en/SpiderMonkey"
-  url "https://archive.mozilla.org/pub/mozilla.org/js/js185-1.0.0.tar.gz"
-  version "1.8.5"
-  sha256 "5d12f7e1f5b4a99436685d97b9b7b75f094d33580227aa998c406bbae6f2a687"
-  revision 2
-
-  head "https://hg.mozilla.org/tracemonkey/archive/tip.tar.gz"
-
-  bottle do
-    cellar :any
-    sha256 "62193341691f6f35a1d844409c587b431aa7540b70c02d90451e2cb3623788de" => :sierra
-    sha256 "5e7789a8ba4e3259364bd3ae827037ba83bf3a076633799bf8f5869b885db399" => :el_capitan
-    sha256 "38d1b7f54b5dbdd4a0e28e3a1077aed2ada42a9266cfaddeda6a08d761a2d8b2" => :yosemite
-  end
-
-  conflicts_with "narwhal", :because => "both install a js binary"
-
-  depends_on "readline"
-  depends_on "nspr"
+  url "https://archive.mozilla.org/pub/firefox/tinderbox-builds/mozilla-release-macosx64/1486087028/jsshell-mac.zip"
```

Comment:
Tap with formulae for ECMAScript/JavaScript engines (downloaded from official sources instead of built from source, for performance reasons): https://github.com/mathiasbynens/homebrew-ecmascript